mindsdb/interfaces/storage/model_fs.py

```diff
@@ -237,26 +252,46 @@ def json_set(self, name, content):
         )
         return json_storage.set(name, content)
 
+    def encrypted_json_set(self, name: str, content: dict) -> None:
+        json_storage = get_encrypted_json_storage(
+            resource_id=self.integration_id,
+            resource_group=RESOURCE_GROUP.INTEGRATION
+        )
+        return json_storage.set(name, content)
+
     def json_get(self, name):
         json_storage = get_json_storage(
             resource_id=self.integration_id,
             resource_group=RESOURCE_GROUP.INTEGRATION
         )
         return json_storage.get(name)
 
+    def encrypted_json_get(self, name: str) -> dict:
+        json_storage = get_encrypted_json_storage(
+            resource_id=self.integration_id,
+            resource_group=RESOURCE_GROUP.INTEGRATION
+        )
+        return json_storage.get(name)
+
     def json_list(self):
         ...
 
     def json_del(self, name):
         ...
 
     def export_files(self) -> bytes:
+        json_storage = self.export_json_storage()
+        if json_storage:
+            json_str = json.dumps(json_storage)
+            self.file_set('json_storage.json', json_str.encode())
```

Comment:
I can suggest a small improvement: 
instead of writing files (with secrets) to FS we can add content directly to zipfile (in line 298):
`zipf.writestr(JSON_STORAGE_FILE, json_str)`
JSON_STORAGE_FILE - is a constant for file with json content